extensions-core/mysql-metadata-storage/src/main/java/io/druid/metadata/storage/mysql/MySQLConnector.java

```diff
@@ -76,6 +76,14 @@ protected String getSerialType()
   }
 
   @Override
+  protected int getStreamingFetchSize()
+  {
+    // this is MySQL's way of indicating you want results streamed back
+    // see http://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-implementation-notes.html
+    return Integer.MIN_VALUE;
```

Comment:
based on the mysql documentation there is no other value that will make it stream results back